src/settings/CountrySelector.ts

```diff
@@ -6,22 +6,50 @@ import type { Country } from "flag-icons";
 import "./CountrySelector.css";
 
 export class CountrySelector {
+    #countries: Country[] = [];
+
     constructor(
         private container: HTMLElement,
         private template: HTMLTemplateElement,
+        private getChecked: () => string[],
```

Comment:
Not sure why we actually need thie `getChecked` callback for this component, and how does it will work together with the [Settings](https://github.com/isqua/flagwall/blob/8ba2444661998db2c58aae74e39df3d41880712f/src/settings/Settings.ts#L23) class that manages the form data.

Reply:
Yes, I missed the logic of the Settings, thank you.